pkg/minikube/node/cache.go

```diff
@@ -154,25 +154,22 @@ func beginDownloadKicBaseImage(g *errgroup.Group, cc *config.ClusterConfig, down
 		var err error
 		for _, img := range append([]string{baseImg}, kic.FallbackImages...) {
 
-			if driver.IsDocker(cc.Driver) && download.ImageExistsInDaemon(img) && !downloadOnly {
-				klog.Infof("%s exists in daemon, skipping load", img)
-				finalImg = img
-				return nil
-			}
+			if driver.IsDocker(cc.Driver) {
+				if download.ImageExistsInDaemon(img) {
+					klog.Infof("%s exists in daemon, skipping load", img)
+					finalImg = img
+					return nil
+				}
 
-			klog.Infof("Downloading %s to local cache", img)
-			err = download.ImageToCache(img)
-			if err == nil {
-				klog.Infof("successfully saved %s as a tarball", img)
-			}
-			if downloadOnly {
-				return nil
-			}
+				klog.Infof("Downloading %s to local cache", img)
+				err = download.ImageToCache(img)
+				if err == nil {
+					klog.Infof("successfully saved %s as a tarball", img)
+				}
+				if downloadOnly {
+					return nil
+				}
 
-			if cc.Driver == driver.Podman {
-				return fmt.Errorf("not yet implemented, see issue #8426")
-			}
```

Comment:
This is the only correct change - removing the bad fmt.Errorf. But it would be more correct to convert this to debug log like:

    Skipping <operation description>: not implemented, see issue #8426

This will improve the user experience but make it easier to debug podman driver, in case skipping has negative effects. If this operation is not needed for podman removing the log is a better change, but I'm not sure what this code is doing and wha is the unimplemented operation that we skipped.

Reply:
done, fwiw.